sdcm/provision/common/utils.py

```diff
@@ -145,26 +145,34 @@ def install_syslogng_service():
                     fi
                 done
 
-                if yum install -y syslog-ng; then
-                    SYSLOG_NG_INSTALLED=1
-                fi
+                for n in 1 2 3; do # cloud-init is running it with set +o braceexpand
```

Comment:
in all these loops (specially the long ones) it would be nicer to use:
`for n in {1..3}`

so in the loop above (line 142) it would be `for n in {1..9}` instead of all these numbers in there

Reply:
There is commend that is describing why it would not work

Reply:
sorry, missed it